tests/strings/strings.cpp

```diff
@@ -845,17 +845,12 @@ void StringTestCase::FromDouble()
     {
         double value;
         int prec;
-        const char *str;
+        const char *str1;
+        const char *str2;
     } testData[] =
     {
         { 1.23,             -1, "1.23" },
-        // All MSVC versions until MSVC 14 used 3 digits for the exponent
```

Comment:
Why did this have to be changed?

Reply:
It fails with MinGW, similar as [this](https://ci.appveyor.com/project/wxWidgets/wxwidgets/build/3387/job/j5a4kaet7bch95nd#L1770).